snap/info.go

```diff
@@ -43,6 +43,15 @@ type PlugInfo struct {
 	Apps      map[string]*AppInfo
 }
 
+// AppNames returns a list of applications names.
+func (plug *PlugInfo) AppNames() []string {
```

Comment:
I just found that to b used a few times in the upcoming code but perhaps the number of times is close to two :-) I'll wake up properly and have a look. We can just make that a private function in the place that needed it.